UNRELEASED.md

```diff
@@ -8,13 +8,19 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - Updated `OptionList` selected styles ([#3633](https://github.com/Shopify/polaris-react/pull/3633))
 - Added the ability to hide the clear filter button on the filter component ([#3049](https://github.com/Shopify/polaris-react/pull/3049))
+- **`Button`:** Right-align `disclosure` when using `textAlignLeft` ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
+- **`Button`:** More consistent transitions ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
+- **`Button`:** New `select` option for `disclosure` ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
```

Comment:
```suggestion
- Right-align `disclosure` when using `textAlignLeft` for `Button` ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
- More consistent transitions in `Button` ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
- New `select` option for `disclosure` in `Button` ([#3709](https://github.com/Shopify/polaris-react/pull/3709))
```

Reply:
Will make the change, but I do think its worth considering this style in the future. We write things this way in `online-store-ui` and I find it much easier to parse the Release Notes and identify the components that are relevant to the consuming app.

As a consumer, when I'm going through `Breaking changes`, I can quickly look at the prefixed components and identify what is relevant to my app.

Reply:
I like this format. I think we should be thinking about these sorts of things in a seperate PR or issue though.